projects/1.19/assets/chat-patches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,60 @@
+{
+    "text.chatpatches.title": "聊天补丁设置",
+
+    "text.chatpatches.dateFormat": "要设置日期和时间格式化样式，点我！",
+    "text.chatpatches.formatCodes": "要设置字符格式化代码和行为，点我！",
+
+    "text.chatpatches.time": "启用时间戳显示",
+    "text.chatpatches.timeDate": "时间戳文本",
+    "text.chatpatches.timeFormat": "时间戳修饰文本",
+    "text.chatpatches.timeColor": "时间戳颜色",
+    "text.chatpatches.desc.time": "是否应该在所有消息前面显示时间戳？",
+    "text.chatpatches.desc.timeDate": "格式化为时间戳的文本。",
+    "text.chatpatches.desc.timeFormat": "使用字符“&” + 用于创建时间戳的格式代码修饰符和修饰文本的格式。字符“$”是上方格式化时间戳选项的占位符。",
+    "text.chatpatches.desc.timeColor": "表示填充的颜色，否则在生成的时间戳中为空。如果要将其与其他格式修饰符一起使用，请在修饰文本选项中使用字符“&r”。",
+
+    "text.chatpatches.hover": "启用悬停显示",
+    "text.chatpatches.hoverDate": "悬停文本",
+    "text.chatpatches.hoverFormat": "悬停修饰文本",
+    "text.chatpatches.hoverColor": "悬停文本颜色",
+    "text.chatpatches.desc.hover": "当您将鼠标悬停在聊天中的时间戳文本上时，是否应该显示更加详细的时间描述文本？",
+    "text.chatpatches.desc.hoverDate": "格式化为较长的日期/时间字符串的文本。",
+    "text.chatpatches.desc.hoverFormat": "使用字符“&” + 用于创建详细悬停说明的格式代码修饰符和修饰文本的格式。需要字符“$”作为上方格式化时间戳选项的占位符。",
```

```suggestion
    "text.chatpatches.desc.hoverFormat": "使用字符“&”+用于创建详细悬停说明的格式化代码修饰符和修饰文本的格式。字符“$”是上方格式化时间戳选项的占位符，不可缺失。",
```
见实际表现，若修改，则下方所有原句表述类似的都需修改